src/merge_label.py

```diff
@@ -111,7 +111,7 @@
 counts.insert(0, 'Label', label)
 #print(counts)
 
-mask=(counts['Barcode'].str.len() == 15)
+mask=(counts['Barcode'].str.len() == 16)  # Scott change - hardcoded barcode length is a bad idea, but so is spending a year refactoring this.
```

Comment:
Oh that's crazy. thanks for the catch! But we should ass teh BC length via the options. In nexflow we knwo them so it should be no problem